dspace-iiif/src/main/java/org/dspace/app/iiif/service/ManifestService.java

```diff
@@ -102,17 +111,57 @@ public ManifestService(ConfigurationService configurationService) {
 
     /**
```

Comment:
I was a little thrown off by the javadoc and needed to read the code. Can the description be changed to say it "will retrieve an customized manifest from the item's manifest bundle or ... if a custom manifest isn't found".